app/src/ep.config.json

```diff
@@ -94,6 +94,38 @@
     "pageId": "<PAGE_ID>",
     "applicationId": "<APP_ID>"
   },
+  "bloomreachSearch": {
+    "enable": false,
+    "config": {
+      "baseUri": "http://core.dxpapi.com/api/v1/core/?",
```

Comment:
Are these all default values, or are they values that should be overwritten as part of the implementation? We will need to add documentation to support this as well as it's brought back into product.

Reply:
These are values based on an Bloomreach environment that was set up for us.  So in a real implementation some of these values should be overwritten.  Sounds good I can help out with the documentation.

Reply:
@aChanEP If these are sensitive values that we have for our environment, we shouldn't be distributing them OOTB in the project. We'll need to document these and backup our own configs somewhere else for future demos, etc.

Reply:
@aChanEP Do we have the `vestri_reference` catalog setup on Bloomreach too? I'm thinking we could remove these values, and add them for our http://reference.elasticpath.com demo instance to show the integration 😄

Reply:
> @aChanEP If these are sensitive values that we have for our environment, we shouldn't be distributing them OOTB in the project. We'll need to document these and backup our own configs somewhere else for future demos, etc.

Yeah that makes total sense to me.  Not quite sure what the process is for adding new documentation.  But perhaps I can begin work on a `BR` integration section that explains these configs and can go along with this PR

Reply:
> @aChanEP Do we have the `vestri_reference` catalog setup on Bloomreach too? I'm thinking we could remove these values, and add them for our http://reference.elasticpath.com demo instance to show the integration 😄

The catalog that should be set up with bloomreach is the one found here:  `vestri-us.epdemos.com/cm/` under the store `vestri_b2c`.  It seems to be the same store as `vestri_reference` found on `http://reference.elasticpath.com` so I'm thinking it should be no problem to add the integration to the existing demo